apps/asap-server/src/handlers/lab/users-handler.ts

```diff
@@ -0,0 +1,82 @@
+import { EventBridgeEvent } from 'aws-lambda';
+import { ListResponse, UserResponse } from '@asap-hub/model';
+import { SquidexGraphql } from '@asap-hub/squidex';
+import {
+  AlgoliaSearchClient,
+  algoliaSearchClientFactory,
+  BatchRequest,
+} from '@asap-hub/algolia';
+import Users, { UserController } from '../../controllers/users';
+import { LabEventType } from '../webhooks/webhook-lab';
+import { algoliaApiKey, algoliaAppId, algoliaIndex } from '../../config';
+import { loopOverCustomCollection } from '../../utils/migrations';
+import logger from '../../utils/logger';
+
+export const indexLabUsersHandler =
```

Comment:
I would refactor this. 

Perhaps you would like to take a look at: 
https://github.com/yldio/asap-hub/blob/622ce9b622db69cec0253b94060c9acfd2c73e55/apps/asap-server/src/handlers/teams/index-handler.ts#L14

It works on a similar basis - for every team that is updated we want to update ROs that are associated with them. However - unlike the team->RO relationship - you will not receive a list of users associated to the lab, so instead of grabbing that from the payload you need to fetch the list of users by the lab - something that you are already doing.